content/settings-template.xml

```diff
@@ -19,7 +19,7 @@
 		<Setting name="Lighting" type="int"> 0 </Setting>
 		<Setting name="GLUseFramebuffer" type="bool"> True </Setting>
 		<Setting name="GLUseNPOT" type="bool"> True </Setting>
-		<Setting name="MouseSensitivity" type="float"> 0.0 </Setting>
+		<Setting name="MouseSensitivity" type="float"> 1.0 </Setting>
```

Comment:
I think this should be left at zero. If I understand the code correctly, we're directly passing the value of the slider to FIFE. The setting in FIFE allows an absolute range of [-0.99 - 10.0], so keeping the default at zero shouldn't change the mouse speed.